app/views/idv/shared/_ssn.html.erb

```diff
@@ -45,6 +56,11 @@ locals:
       html: { autocomplete: 'off' },
     ) do |f| %>
   <div class="tablet:grid-col-8">
+    <% if IdentityConfig.store.proofing_device_profiling_collecting_enabled %>
+      <% unless IdentityConfig.store.lexisnexis_threatmetrix_account_id.empty? || updating_ssn %>
+        <%= f.hidden_field :threatmetrix_session_id, value: threatmetrix_session_id %>
```

Comment:
For security/assurance purposes, what if we persisted this ID on the backend, in the session somewhere, instead of printing it to the page?